org.sugarlabs.MazeActivity.json

```diff
@@ -0,0 +1,266 @@
+
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.MazeActivity",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/Maze.activity"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "empy",
+            "cleanup": [ "*" ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} empy"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/3b/95/88ed47cb7da88569a78b7d6fb9420298df7e99997810c844a924d96d3c08/empy-3.3.4.tar.gz",
+                    "sha256": "73ac49785b601479df4ea18a7c79bc1304a8a7c34c02b9472cf1206ae88f01b3"
+                }
+            ],
+            "post-install": [
+                "chmod +x /app/lib/python3.7/site-packages/em.py",
+                "mkdir -p /app/bin/",
+                "ln -s /app/lib/python3.7/site-packages/em.py /app/bin/empy"
+            ]
+        },
+        {
+            "name": "perl-XML-Simple",
+            "cleanup": [ "*" ],
+            "no-autogen": true,
+            "build-options": {
+                "no-debuginfo": true,
+                "env": "/app/lib/perl5/site_perl/5.30.0/"
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-Simple-2.25.tar.gz",
+                    "sha256": "531fddaebea2416743eb5c4fdfab028f502123d9a220405a4100e68fc480dbf8"
+                },
+                {
+                    "type": "file",
+                    "path": "perl-MakefilePL-Makefile",
+                    "dest-filename": "Makefile"
+                }
+            ]
+        },
+        {
+            "name": "xcursorgen",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/freedesktop/xcursorgen/archive/xcursorgen-1.0.6.tar.gz",
+                    "sha256": "0566c6848c93ef092b37e4ffdc6e86e9249fa9721f663117787b6cefea5eb55b"
+                }
+            ]
+        },
+        {
+            "name": "icon-naming-utils",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.gz",
+                    "sha256": "044ab2199ed8c6a55ce36fd4fcd8b8021a5e21f5bab028c0a7cdcf52a5902e1c"
+                }
+            ]
+        },
+        {
+            "name": "intltool",
```

Comment:
We can do a little improvement here, so te reduce manifest length:

1. include the shared-modules git submodule, `git submodule add https://github.com/flathub/shared-modules.git`
2. reference the submodules in the manifest https://github.com/tchx84/org.sugarlabs.Chart-flathub/blob/master/org.sugarlabs.Chart.json#L24-L25
3. commit the changes.
`